.github/workflows/claude.yml

```
@@ -10,15 +11,54 @@ on:
     types: [submitted]
 
 jobs:
-  claude-response:
+  claude:
+    if: |
```

actionlint (1.7.7)

got unexpected character '#' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)